src/main/core/Controller/APINew/User/UserController.php

```diff
@@ -277,6 +281,7 @@ public function disableUsersAction(Request $request): JsonResponse
         foreach ($users as $user) {
             if ($user->isEnabled() && $this->checkPermission('ADMINISTRATE', $user)) {
                 $this->manager->disable($user);
+                $this->eventDispatcher->dispatch(SecurityEvents::USER_DISABLE, UserDisableEvent::class, [$user]);
```

Comment:
Should be dispatched from the `disable()` method. Not in the controller.

Reply:
Done, thx :)